internal/databox/static/scaffoldings/golang/go.mod

```diff
@@ -1,6 +1,6 @@
 module [[.Component]]
 
-go 1.19
+go 1.18
```

Comment:
Just curious on why go.1.18?

Reply:
Because of https://github.com/lacework/go-sdk/blob/main/go.mod#L3